docs/api/platform/Wait.md

```diff
@@ -0,0 +1,18 @@
+# Wait
+
+<span class="tips">**Tip:** `wait` is deprecated in favor of explicit sleep functions. To sleep, replace `wait` with `ThisThread::sleep_for` (C++) or `thread_sleep_for` (C). To wait (without sleeping), call `wait_us`. `wait_us` is safe to call from ISR context.</span>
```

Comment:
It i smore than a tip?
`wait` is not only depreacted but also removed in Mbed 6

Reply:
Didn't we just put it back? Did I misunderstand PR https://github.com/ARMmbed/mbed-os-5-docs/pull/1306?

Reply:
Or am I confusing a class with a function/method?

Reply:
We have remove the `wait` function but we have kept the `wait_us` and `wait_us` functions.